yarn-project/simulator/src/public/executor.ts

```diff
@@ -66,6 +66,8 @@ async function executePublicFunctionAvm(executionContext: PublicExecutionContext
     executionContext.commitmentsDb,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 55]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Maybe add a comment clarifying that this function is (as opposed to ...Acvm) only called at the top level but not for nested calls.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/6165?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Will do (in the next PR though where that is actually made true)

Reply:
Oh I was confused, that's this PR :+1: